include/z64actor.h

```diff
@@ -164,8 +164,10 @@ typedef struct ActorShape {
 //
 #define ACTOR_FLAG_16 (1 << 16)
 
-//
-#define ACTOR_FLAG_17 (1 << 17)
+// Actor will be influenced by the pitch (x rot) of Player's left hand when being carried.
+// This is different from the default actor carry behavior, which influences the actor's yaw.
```

Comment:
```suggestion
// Actor will inherit the pitch (x rot) of Player's left hand when being carried,
// instead of the yaw as with the default actor carry behavior.
```
Your call about using "inherit" or not
The point is more I think this could be clearer about the yaw no longer being set under this flag

Reply:
sounds good to me
(I didn't notice it was in fact not a simple assignment)

I'd just move the comma and add a "the" if that's alright:
```
// Actor will be influenced by the pitch (x rot) of Player's left hand when being carried,
// instead of the Player's yaw which is the default actor carry behavior.
```

Reply:
The natural pause to me in the sentence is where I have it now. but yeah doesnt matter too much to me I guess.

As for "the", I only find that necessary when referring to the person controlling Link "the player", vs just referring to Player the actor "Player's yaw"

Reply:
I'll leave it up to you then

Reply:
moved the comma but yeah ill keep "the" out